### DISCUSSION TITLE
A = Channel Name, B = Frequency #

### DISCUSSION THREAD

**Discussion by MikeGriffinReborn** [egzumer/uv-k5-firmware-custom#297] [General]
Is there any way to have the A "band" show say the "Channel Name" while the "B" band shows the "Frequency" 

I love this feature on the BaoFengs as it helps me learn to match up frequency's with names

**Comment by egzumer** [comment]
No, there is only one setting for both, but you can turn on Name+Freq for both

**Reply by MikeGriffinReborn** [reply]
I see that you can actually set that via the Quansheng menu itself, but neither Chirp nor Quanshang's Programming software have "both" as an option. I set it last night after your response on the Radio itself and love it. By the way, the way the menu's are setup are awesome, I played around with the Spectrum Analyzer last night and it was awesome, once you figure out the keys that control it it's simply incredible that this little guy can do. imagine the radio's that will be coming out in two years !

Thank you for this incredible software, as a developer myself that has done tones of "FreeWare" and other software on my own time I kow what a labor of love it is, see [ https://github.com/MikeGriffinReborn/EntitySpaces]( https://github.com/MikeGriffinReborn/EntitySpaces) simply scroll down. I see you're using C, I did C for a long time then 10+ years of C++, now C# for 20 years (ya, since 2000). Again, thank yoiu

**Reply by egzumer** [reply]
It looks like you didn't read https://github.com/egzumer/uv-k5-firmware-custom/wiki.

I'm more of a C++ programmer actually (none embedded). My first opensource work that I've put that much time to.

**Comment by ErikS-web** [comment]
With the [dedicated chirp driver ](https://github.com/egzumer/uvk5-chirp-driver)you can configure it also on the PC.

![Knipsel](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/d3a33be3-0233-48d4-8a0c-43e227dd6069)

But for both A and B channels simultaneously with Menu 33 `channel display style` set to "Name + Freq" 
and after changing it to Both, I also think its ideal.
![NameAndFreq](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/443b2128-d4bd-44c0-ac5e-b93c33e86595)

_Thanks for asking. I now see a _type-error_ (letter missing)_ _Will be fixed._
